Source/WebCore/accessibility/AccessibilitySVGObject.cpp

```diff
@@ -247,6 +241,11 @@ bool AccessibilitySVGElement::computeIsIgnored() const
         }
     }
 
+    if (auto* svgInlineText = dynamicDowncast<RenderSVGInlineText>(m_renderer.get())) {
+        if (svgInlineText->containsOnlyCollapsibleWhitespace())
```

Comment:
Do we have a test for this change?

Reply:
We don't. And I tried to craft one after your comment, but I couldn't, despite creating SVG text with what should be collapsible whitespace. So I removed this until it's proven necessary by something we can actually capture in markup.